apps/developer-hub/src/components/BlankPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { identity } from '@deso-core/identity';
```

Comment:
this file is just a dummy page to show what using this flow might look like for developers.